examples/quickstart/jupyter-notebooks/notebooks/03-query/03-approxCountDistinct.ipynb

```diff
@@ -0,0 +1,470 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "557e06e8-9b35-4b34-8322-8a8ede6de709",
+   "metadata": {},
+   "source": [
+    "# Counting distinct values\n",
+    "\n",
+    "__It's extremely common for analysts to want to count unique occurences of some dimension value in data. With the Druid database's history of large volumes of data comes an advanced computer science technique to speed up this calculation through approximation. In this tutorial, work through some examples and see the effect of turning it on and off, and of making it even faster by pre-generating the objects that Druid uses to execute the query.__\n",
```

Comment:
Why is this bold?

Reply:
Habit as a blog intro. Removed the boldness.